pkg/cluster/context.go

```diff
@@ -178,6 +178,15 @@ func (c *Context) Delete() error {
 	if err != nil {
 		return fmt.Errorf("error listing nodes: %v", err)
 	}
+
+	// clean up the kind kube config file that was created earlier
+	// also let the user know that they need to unset $KUBECONFIG to
+	// go back to their normal contexts
```

Comment:
I think this could be changed to something like:

> // Remove the kind kube config file created by "kind create cluster."
> // If $KUBECONFIG was set to the kubeconfig file created by kind, then remind user to unset its value.

and 
```go
fmt.Println("kind kubectl config file was removed.\nRemeber to unset $KUBECONFIG, if set")
```